File: test2/src/main/java/xyz/yggdrazil/delaunay/voronoi/nodename/as3delaunay/Winding.java
package xyz.yggdrazil.delaunay.voronoi.nodename.as3delaunay;

public final class Winding {

    final public static Winding CLOCKWISE = new Winding("clockwise");
    final public static Winding COUNTERCLOCKWISE = new Winding("counterclockwise");
    final public static Winding NONE = new Winding("none");
    private String name;

    private Winding(String name) {
        super();
        this.name = name;
    }

    @Override
    public String toString() {
        return name;
    }
}

File: test2/src/main/java/xyz/yggdrazil/delaunay/voronoi/nodename/as3delaunay/Winding.kt
package xyz.yggdrazil.delaunay.voronoi.nodename.as3delaunay

class Winding private constructor(private val name: String) {

    override fun toString(): String {
        return name
    }

    companion object {

        val CLOCKWISE = Winding("clockwise")
        val COUNTERCLOCKWISE = Winding("counterclockwise")
        val NONE = Winding("none")
    }
}